sensor_msgs/LaserScan laser_in
---
geometry_msgs/Point point_a
geometry_msgs/Point point_c

